src/UICrafter.Client/Components/ComponentProperties.razor

```diff
@@ -0,0 +1,109 @@
+@using static UICrafter.Client.Pages.UICrafterPage
+
+<MudStack Style="width: 25%;">
+    <MudText>Properties:</MudText>
+
+    <MudStack Class="rounded mud-background-gray pa-4 my-2 ">
+        @if (SelectedComponent is not null)
```

Comment:
Depending on the size, and how many components we should consider doing an inhertiance structure and encapsulating these in seperate individual components.

Reply:
Enig